cmd/bosun/conf/system.go

```diff
@@ -82,6 +84,7 @@ func (sc *SystemConf) EnabledBackends() EnabledBackends {
 	b.Influx = sc.InfluxConf.URL != ""
 	b.Elastic = len(sc.ElasticConf["default"].Hosts) != 0
 	b.Annotate = len(sc.AnnotateConf.Hosts) != 0
+	b.CloudWatch = true
```

Comment:
I don't thing global true is the right default here. It needs to check that cloudwatch is actually enabled.

Reply:
Fixed